### PR TITLE
[1.5.0] Version modules for release

### PR DESCRIPTION
**Pull request by cormacpayne**
## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by cormacpayne** [comment]
Sign job: https://azuresdkci.westus2.cloudapp.azure.com/job/psnetsign-esrp/204/

**Comment by maddieclayton** [comment]
This iteration looks good to me.

**Comment by cormacpayne** [comment]
Sign job: https://azuresdkci.westus2.cloudapp.azure.com/job/psnetsign-esrp/210/

File: tools/Az/Az.psd1 [hunk 1]
```diff
@@ -153,71 +153,50 @@ PrivateData = @{
         # IconUri = ''
 
         # ReleaseNotes of this module
-        ReleaseNotes = '1.4.0 - February 2019
-Az.AnalysisServices
-* Deprecated AddAzureASAccount cmdlet
+        ReleaseNotes = '1.5.0 - March 2019
+Az.Accounts
+* Add ''Register-AzModule'' command to suppoirt AutoRest generated cmdlets
```
Comment (markcowl) [inline tools/Az/Az.psd1]: support

**Comment by cormacpayne** [comment]
Sign job: https://azuresdkci.westus2.cloudapp.azure.com/job/psnetsign-esrp/211/

**Pull request merged**